pkg/base1/cockpit.js

```diff
@@ -1162,7 +1162,7 @@ function full_scope(cockpit, $, po) {
         else
             path = "" + path;
         var options = { command: "jump", location: path };
-        if (host)
+        if (host !== undefined)
```

Comment:
Technically this if statement makes no difference. ```JSON.stringify``` skips object values that are undefined.